samples/security/psp/citadel-agent-psp.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: istio-nodeagent
+spec:
+  allowedHostPaths:
+    - pathPrefix: "/var/run/sds"
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  fsGroup:
+    rule: RunAsAny
+  volumes:
+    - '*'
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: istio-nodeagent
```

Comment:
Can we make this role more specific?
Like istio-nodeagent-istio-system-psp
I think our role names are following the rule like `<component name>-<namespace>`.